### PR TITLE
Add render views for datatables [SCI-4033]

### PR DESCRIPTION
**Pull request by aignatov-bio**
Jira ticket: [SCI-4033](https://biosistemika.atlassian.net/browse/SCI-4033)

### What was done
_In a sentence or two describe what was done, focus on the tech aspect_

### CODE REVIEW

File: app/assets/javascripts/application.js.erb [hunk 1]
```diff
@@ -42,8 +42,9 @@
 //= require select2_customization
 //= require shared/inline_editing
 //= require activestorage
-//= require turbolinks
 //= require marvinjslauncher
+//= require_directory ./repositories/renders
```
Comment (okriuchykhin) [inline app/assets/javascripts/application.js.erb]: isn't it working without './'?

and I think 'renderers' is more correct name

File: app/helpers/repository_datatable_helper.rb [hunk 2]
```diff
@@ -71,29 +72,29 @@ def default_table_columns
...
-    if (DisplayCellValue.respond_to? value_type) && defined?(render)
-      render DisplayCellValue.public_send(value_type, cell, team)
-    elsif DisplayCellValue.respond_to? "#{value_type}_text"
-      DisplayCellValue.public_send("#{value_type}_text", cell, team)
-    else
-      custom_auto_link(display_tooltip(cell.value.data, Constants::NAME_MAX_LENGTH),
-                       simple_format: true,
-                       team: team)
-    end
+    cell_type = cell.value_type.underscore
```
Comment (okriuchykhin) [inline app/helpers/repository_datatable_helper.rb]: We can create separate serializers and call them by cell.value_type. Each data type should have implemented serializers, so there is no need to check if method present

File: app/assets/javascripts/application.js.erb (hunk 1 above)
Comment (aignatov-bio) [reply app/assets/javascripts/application.js.erb]: Fixed

File: app/helpers/repository_datatable_helper.rb (hunk 2 above)
Comment (aignatov-bio) [reply app/helpers/repository_datatable_helper.rb]: I created serializes for exist  column types

**Pull request merged**